src/context/index.jsx

```diff
@@ -1,15 +1,46 @@
-import React, { createContext, useState } from 'react'
+import React, { createContext, useState, useEffect } from 'react'
 import PropTypes from 'prop-types'
+import ChatController from 'networking/controllers'
 
 const WidgetContext = createContext()
 
 const WidgetContextProvider = ({ children }) => {
   const [open, setOpen] = useState(false)
   const toggleWidget = () => setOpen(!open)
-  const value = { open, toggleWidget }
+  const [loading, setLoading] = useState(true)
+  const [personalization, setPersonalization] = useState({
+    title: 'Hello 👋',
+    subtitle: 'tell us about your problem',
+    mainIcon: 'https://reactjsexample.com/favicon.png',
+    launcherIcon: undefined,
+    position: 'top-right', // 'top-left', 'top-right', 'bottom-left', 'bottom-right',
+  })
+
+  const getPersonalization = async () => {
+    try {
+      const response = await ChatController.getPersonalization()
+      const newPersonalization = { ...personalization }
+      Object.keys(response.data).forEach(key => {
+        if (response.data[key]) newPersonalization[key] = response.data[key]
+      })
+      setPersonalization(newPersonalization)
+    } catch (error) {
+      console.log(error)
+    }
+    setLoading(false)
+  }
+
+  useEffect(() => {
```

Comment:
Making a function call instead of the async one would remove the need to disable the linter.
```suggestion
  const handlePersonalizationCall = () => getPersonalization()
  useEffect(() => {
    handlePersonalizationCall()
  }, [])
```

Reply:
Doesn't work either 😞